nestedtensor/nested/nested.py

```diff
@@ -213,11 +213,104 @@ def nested_stride(self, dim=None):
     def __torch_function__(self, func, types, args=(), kwargs=None):
         _local_func = None
         if func in NestedTensor.__C_functions:
+            if 'interpolate' in str(func):
+                # size can be None, torch.Size, int or tuple of int
```

Comment:
None of this should be necessary.

Reply:
We discussed this and decided to cover all the cases. Seg pipeline calls with 2 different inputs. Also, the work is already done.